dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -374,6 +374,29 @@ var playerSet = (function () {
       return distances[0].player;
     };
 
+    PlayerSet.prototype.getAdjacentPlayers = function () {
+      /* Return a list of players adjacent to the ego player */
+      adjacentPlayers = [];
+      let egoPostiion = this._players[ego].position;
+      for (id in this._players) {
```

Comment:
I think we could modernize a bit and reduce some noise:
```
for (const [id, player] of this._players.entries()) {
...
``` 

Though this would be different from some of the existing code, it might point in the direction we want to move in.

Not passionate about this... just a thought.